### PR TITLE
[mobile] Add padding to Date picker (Calendar view)

### PR DESCRIPTION
**Pull request by DohyungLim**
- padding: 14px 16px added.
- date underline removed.
- when screen width: 361+, fix width 328px.

### CODE REVIEW

**Comment by DohyungLim** [comment]
Snapshot :

image

image

image

File: examples/mobile/UIComponents/components/dialogs/pickers/calendar.html [hunk 1]
```diff
@@ -18,7 +18,9 @@ <h1>
 		</div>
 		<div class="ui-content">
 			<ul class="ui-listview ui-content-area">
-				<div class="ui-calendar" data-past-selection="true"></div>
+				<div class="ui-calendar-container">
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/dialogs/pickers/calendar.html]: Why are you adding the "ui-calendar-container""?
You can apply the style without container.

Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/pickers/calendar.html]: I removed div.
and add style to padding in ui-content-area.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**